docker-compose.yml

```diff
@@ -675,6 +845,10 @@ services:
       - static
       - diet-error-handler
     environment:
+      MEMCACHE_SERVERS: memcached
```

Comment:
I notice there isn't a link or depends_on for this so I'm wondering if this is actually able to communicate with the memcached container?

Reply:
Good catch, we'll add the dependency in.  Because we run ` make up ` all the services will be up anyway, which is probably why we missed it. 🤦‍♂️ 

>Links allow you to define extra aliases by which a service is reachable from another service. They are not required to enable services to communicate - **by default, any service can reach any other service at that service’s name.** In the following example, db is reachable from web at the hostnames db and database:

https://docs.docker.com/compose/networking/#links

Reply:
Ahh TIL - Thanks 👍